api/proto/config/edgeview.proto

```diff
@@ -0,0 +1,50 @@
+// Copyright(c) 2021 Zededa, Inc.
+// All rights reserved.
+
+syntax = "proto3";
+
+package org.lfedge.eve.config;
+option go_package  = "github.com/lf-edge/eve/api/go/config";
+option java_package = "org.lfedge.eve.config";
+
+message EdgeViewConfig {
+    // JWT token for signed info, it contains the dispatcher
+    // endpoint IP:Port, device UUID and expiration time
+    string  token = 1;
+    // dispatcher certificate(s) if it's not well-known CA signed
+    repeated bytes disp_cert_pem = 2;
+    // policy for device access through edge-view
+    DevEdgeViewPolicy dev_policy = 3;
```

Comment:
Great that you have the ssh policy in there. But this means this isn't specific to edge view but should be placed somewhere else in the EdgeDevConfig. Would a name like DebugAccessPolicy containing a DevDebugAccessPolicy and an AppDebugAccessPolicy make sense?

Reply:
@eriknordmark my intention for this policy dev and app being only for the edge-view operation. we can later add the other operational policies for the device level alter.
Sure I'll rename them to use the 'DebugAccess' wording.

Reply:
My problem with that approach is that you have to first document the current independent approach and some folks might read that documentation, and later change the documentation to specify the combined policy.
Thus deferring this only makes sense if we are pretty certain that nobody will use this until we have done the combined work, hence we will defer documentation and talking about this feature as "done" until that point in time.